base/bin/migration.mustache

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const mustache = require('mustache');
+
+const now = new Date().toISOString();
+const migrationName = now + '_' + process.argv[2];
+
+const opts = {migrationName: migrationName};
+
+const migrationTemplate = fs.readFileSync('./bin/migration_template.mustache').toString();
+const migrationTestTemplate = fs.readFileSync('./bin/migration_test_template.mustache').toString();
```

Comment:
This `fs.readFileSync({template}).toString()` is duplicated around the app a bit - you could create `/lib/template` and add a `stringify(tpl)` method to reduce this.